src/backend/distributed/commands/table.c

```diff
@@ -71,53 +75,97 @@ PreprocessDropTableStmt(Node *node, const char *queryString)
 
 	Assert(dropTableStatement->removeType == OBJECT_TABLE);
 
+	List *citusTableOids = ExtractCitusTableOidsFromDropStmt(dropTableStatement);
```

Comment:
Do we have a convention for Oids? Oid vs Id?

Reply:
Minor: Also sometimes I see item -> items and sometimes item -> itemList. Our convention is probably the second one.

Reply:
I prefer to use `Oid` in the most contexts to distinguish it from other ids like shardId etc.

Using `List` sounds good to me, I should change it to that